frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventResultDetails/Modules/QuizResultDetails/QuizResultViews/DialogUtils/QuizResultDetailsDialog.tsx

```diff
@@ -0,0 +1,190 @@
+import "./QuizResultDetailsDialog.scss";
+
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Paper,
+} from "@material-ui/core";
+import TotalAreaChart, {
+  TotalAreaChartData,
+} from "../../../../../../SessionResultSummary/TotalAreaChart";
+
+import AppBar from "@material-ui/core/AppBar";
+import CloseIcon from "@material-ui/icons/Close";
+import Dialog from "@material-ui/core/Dialog";
+import EnhancedTable from "../../../../../../../utils/Table/Table";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import IconButton from "@material-ui/core/IconButton";
+import React from "react";
+import Slide from "@material-ui/core/Slide";
+import Toolbar from "@material-ui/core/Toolbar";
+import { TransitionProps } from "@material-ui/core/transitions";
+import Typography from "@material-ui/core/Typography";
+import { useTranslation } from "react-i18next";
+
+export interface QuizResultDetailsDialogProps {
+  isOpen: boolean;
+  handleClose: () => void;
+  title: string;
+  detailName: string;
+  detailIcon: React.ReactElement;
+  stats: {
+    label: string;
+    value: string;
+  }[];
+  detailsViewTitle: string;
+  details: {
```

Comment:
Extract type for details, probably same goes for trackNodes and stats. This whole props interface is too big.